Assets/_BForBoss/_Core/Scripts/Editor/LevelDesignFeedback/LevelDesignFeedbackEditorWindow.cs

```diff
@@ -0,0 +1,213 @@
+using System;
+using System.Text;
+using BForBoss;
+using Trello;
```

Comment:
So I don't know how coupled you want this to be with Trello, but basically, this whole LevelDesignFeedback won't be able to run without Trello.

You should at the very least decouple the trello logic from the "editorWindow" view.

Reply:
Normally I would agree, like create a new script that gets called on `SendFeedback()`, who's responsibility is to connect to Trello, but I don't really see the benefit of going through the trouble for the sake of  EditorCode, i.e things that won't be published for the game? 

Do you still think its worth?

Reply:
Yeah, cuz this is iteration 1, and it should be setup for easier code changes.

Reply:
Normally i wouldn't really mind, but this looks like quite a few components. A whole flow frontend to backend. So it's better to make it easier to read/parse and understand for someone else.